sda-commons-dependencies/build.gradle

```diff
@@ -83,6 +84,7 @@ dependencies {
     api "io.opentracing.contrib:opentracing-web-servlet-filter:0.4.1"
     api "org.jetbrains.kotlin:kotlin-stdlib:1.3.72" // override version from dropwizard-bom
 
+    api "javax.validation:validation-api:2.0.1.Final"
```

Comment:
This clashes with the old version of hibernate validator which we can't upgrade in Dropwizard, as far as I know. Therefore we always added this version as a test dependency only in the project. There is a high risk that someone uses the new validation-api annotations in production code. That would only have an effect on the documentation but won't trigger a validation with hibernate-validator. There are some annotations that the old hibernate version provides itself which do not exist in 1.x of the validation-api. Right now I do not have a good solution for this. It may be needed to register a validation class for the new annotations in hibernate validator manually.

Reply:
I don't think I can downgrade it as it will run into an `java.lang.NoClassDefFoundError: javax/validation/constraints/NotBlank` error 😢 

Do you know about this? Do we have tests for this?

Reply:
Looks like this would be fixed with Dropwizard two. Maybe we make this an experimental feature, document it so that users have to override the dependency in the consuming package and know about the risks. With a Dropwizard 2 upgrade we can make it the default.

Reply:
We fixed this by reverting to the old version and include the missing classes in our code.